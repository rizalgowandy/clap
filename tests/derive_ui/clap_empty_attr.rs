use clap::Parser;

#[derive(Parser, Debug)]
#[command]
struct Opt {}

#[derive(Parser, Debug)]
struct Opt1 {
    #[arg = "short"]
    foo: u32,
}

fn main() {
    let opt = Opt::parse();
    println!("{opt:?}");
}
