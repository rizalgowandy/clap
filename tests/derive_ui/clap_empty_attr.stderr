error: expected attribute arguments in parentheses: #[command(...)]
 --> tests/derive_ui/clap_empty_attr.rs:4:3
  |
4 | #[command]
  |   ^^^^^^^

error: expected parentheses: #[arg(...)]
 --> tests/derive_ui/clap_empty_attr.rs:9:11
  |
9 |     #[arg = "short"]
  |           ^
